js/index.js

```diff
@@ -40,3 +40,89 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+///Match Index w/ Original File
+
+const navbar = document.querySelectorAll('nav a')
+
+navbar[0].textContent = siteContent['nav']['nav-item-1'];
```

Comment:
Great job with the nav items. Lets try to practice getting things done in loops though.